sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/EmailChannelProperties.cs

```diff
@@ -31,12 +31,12 @@ public EmailChannelProperties()
         /// Initializes a new instance of the EmailChannelProperties class.
         /// </summary>
         /// <param name="emailAddress">The email address</param>
+        /// <param name="isEnabled">Whether this channel is enabled for the
+        /// bot</param>
         /// <param name="password">The password for the email address. Value
         /// only returned through POST to the action Channel List API,
         /// otherwise empty.</param>
-        /// <param name="isEnabled">Whether this channel is enabled for the
-        /// bot</param>
-        public EmailChannelProperties(string emailAddress, string password, bool isEnabled)
+        public EmailChannelProperties(string emailAddress, bool isEnabled, string password = default(string))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
added fallback constructor in Customization/EmailChannelProperties.cs as partial class